Makefile

```diff
@@ -152,27 +149,30 @@ ELF := $(ROM:.z64=.elf)
 # description of ROM segments
 SPEC := spec
 
-ifeq ($(COMPILER),ido)
-SRC_DIRS := $(shell find src -type d -not -path src/gcc_fix)
-else
-SRC_DIRS := $(shell find src -type d)
+# Folders ZAPD extracts assets to
+ASSETS_ZAPD_DEST_DIRS := assets/code assets/misc assets/objects assets/overlays assets/scenes assets/textures
```

Comment:
Can this simply be `ASSET_DIRS`?

Reply:
Not really, it's specifically the ones ZAPD touches (or we can find a better name to refer to this group of assets but idk)
For example it leaves out text, and in the future audio stuff too